submissions/AndriiBabushko/html-css_popup/css/style.css

```diff
@@ -0,0 +1,283 @@
+@import url("https://fonts.googleapis.com/css2?family=Space+Mono&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Dancing+Script&display=swap");
+
+@keyframes nav-animation {
+    from {
+        transform: scale(1);
+    }
+    50% {
+        transform: scale(1.2);
+        filter: invert(54%) sepia(4%) saturate(14%) hue-rotate(1deg) brightness(93%) contrast(91%);
+    }
+    to {
+        transform: scale(1);
+    }
+}
+
+@keyframes logo-animation {
+    from {
+        transform: scale(1);
+    }
+    50% {
+        transform: scale(1.2);
+    }
+    to {
+        transform: scale(1);
+    }
+}
+
+@keyframes one_time_180_rotate {
+    from {
+        transform: rotate(0deg);
+    }
+    to {
+        transform: rotate(180deg);
+    }
+}
+
+@keyframes infinite_180_rotate {
+    from {
+        transform: rotate(0deg);
+    }
+    50% {
+        transform: rotate(180deg);
+    }
+    to {
+        transform: rotate(0deg);
+    }
+}
+
+* {
+    padding: 0;
+    margin: 0;
+}
+
+*::-webkit-scrollbar {
+    width: 10px;
+}
+
+*::-webkit-scrollbar-thumb {
+    background-color: grey;
+}
+
+a {
+    cursor: pointer;
+}
+
+body {
+    overflow-x: hidden;
+}
+
+/* Page wrapper */
+.wrapper {
+    box-sizing: border-box;
+    font-family: OpenSans-Regular, "Times New Roman", Times, serif;
+}
+
+.wrapper .header {
+    background-color: #DBC7B5;
+    height: 65px;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100vw;
+    z-index: 1;
+    box-shadow: 0 3px 15px 1px #735244;
+}
+
+.wrapper .header .header__container {
```

Comment:
Avoid too specific CSS selectors. [Read about that](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

Reply:
✓